README.md

```diff
@@ -26,25 +26,39 @@ Each of these pipeline definitions does four things:
 pool:
   vmImage: ubuntu-latest
 steps:
-  - task: InstallMATLAB@0
-  - task: RunMATLABTests@0
+  - task: InstallMATLAB@1
+  - task: RunMATLABTests@1
     inputs:
       sourceFolder: source
       codeCoverageCobertura: coverage.xml
-  - script: bash <(curl -s https://codecov.io/bash)
+  - script: |
```

Comment:
The original code was terse. This replacement is quite verbose and hard to understand. Can I know why we had to make this replacement?

Reply:
It looks like the bash script is still available, but they are moving towards using their CLI going forward. When you setup a new project on codecov.io and choose "Other CI" (Not GitHub Actions or CircleCI) they recommend this code:
<img width="786" alt="Screenshot 2024-04-02 at 11 10 25 AM" src="https://github.com/mathworks/matlab-codecov-example/assets/103441853/b21f8e4b-d35d-4c76-91f0-c5bc57c1011e">